src/server/server.js

```diff
@@ -107,6 +107,51 @@ const createStock = quote => {
   };
 };
 
+const calculateDateYearsInPast = years => {
+  const year = new Date().getFullYear() - years;
+  const date = new Date();
+  date.setFullYear(year);
+  return date;
+};
+
+const padSingleDigitWithZero = value => {
+  if (typeof value !== 'number' && typeof value !== 'string') {
+    throw new Error(`${padSingleDigitWithZero.name} requires a number or string`);
+  }
+  let num = parseInt(value);
+  return num < 10 ? '0' + num : num.toString();
+};
+
+const formatDate = date => {
```

Comment:
I just had a thought... Why do we even have this function? Can't we use the dateFormat library we installed for this? Then we don't even need to write tests for this, and we avoid reinventing the wheel.